pkg/delete_orga.go

```diff
@@ -0,0 +1,56 @@
+package pkg
+
+import (
+	"fmt"
+	"github.com/qovery/qovery-cli/utils"
+	log "github.com/sirupsen/logrus"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+)
+
+func DeleteOrganizationByClusterId(clusterId string, dryRunDisabled bool) {
+	utils.CheckAdminUrl()
+
+	utils.DryRunPrint(dryRunDisabled)
+	if utils.Validate("delete") {
+		res := delete(os.Getenv("ADMIN_URL")+"/organization/"+clusterId, http.MethodDelete, dryRunDisabled)
```

Comment:
The path shouldn't be `/organization/:clusterId` but more something like `/organization?cluster_id=:clusterId`.  Or maybe send `clusterId` in the request payload

As it is now it's confusing because you would thing (at first glance) that you need an `organizationId` to make this request work

Reply:
Well, I fit to endpoint made in core.

Reply:
https://gitlab.com/qovery/backend/q-core/-/merge_requests/433